Formula/libmxnet.rb

```diff
@@ -0,0 +1,33 @@
+class Libmxnet < Formula
+  desc "Deep Learning with Dynamic, Mutation-aware Dataflow Dep Scheduler"
+  homepage "https://mxnet.apache.org"
+  url "https://github.com/apache/incubator-mxnet.git",
+      tag:      "v2.0.0.beta0.rc1",
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
Can one do "head-only" then or no?

The reasoning behind the beta here is twofold: 
- 2.x release seems to be nearing upstream
- 1.x release has outdated 3rd party deps (mainly onednn) that will require additional unnecessary work to get it work (and there is no reason to bother with it if 2.x is coming soon)

Also, any other feedback if I were to resubmit this again?

Reply:
Head-only is still not a stable release so unfortunately no.

Reply:
Thanks! I will consider submitting this again in the future :)